tracer/tracer.go

```diff
@@ -73,6 +74,10 @@ func NewTracerTransport(transport Transport) *Tracer {
 		exitWG: &sync.WaitGroup{},
 	}
 
+	// Add some info about the language used
+	t.SetMeta("lang", "go")
+	t.SetMeta("lang.version", runtime.Version())
+
```

Comment:
Here is the real modification: we add 2 new meta in the default tracer: `lang` and `lang.version`.